-- Journal begins at Fri 2022-06-03 06:08:08 UTC, ends at Fri 2022-06-03 06:08:57 UTC. --
Jun 03 06:08:08 kind-worker systemd[1]: Condition check resulted in kubelet: The Kubernetes Node Agent being skipped.
Jun 03 06:08:41 kind-worker systemd[1]: Starting kubelet: The Kubernetes Node Agent...
Jun 03 06:08:41 kind-worker systemd[1]: Started kubelet: The Kubernetes Node Agent.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --container-runtime has been deprecated, will be removed in 1.27 as the only valid value is 'remote'
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --pod-infra-container-image has been deprecated, will be removed in 1.27. Image garbage collector will get sandbox image information from CRI.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:41 kind-worker kubelet[298]: I0603 06:08:41.519561     298 server.go:193] "--pod-infra-container-image will not be pruned by the image garbage collector in kubelet and should also be set in the remote runtime"
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --container-runtime has been deprecated, will be removed in 1.27 as the only valid value is 'remote'
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --pod-infra-container-image has been deprecated, will be removed in 1.27. Image garbage collector will get sandbox image information from CRI.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --provider-id has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:41 kind-worker kubelet[298]: Flag --cgroup-root has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.026281     298 server.go:399] "Kubelet version" kubeletVersion="v1.24.0"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.026315     298 server.go:401] "Golang settings" GOGC="" GOMAXPROCS="" GOTRACEBACK=""
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.026755     298 server.go:813] "Client rotation is on, will bootstrap in background"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.030235     298 dynamic_cafile_content.go:157] "Starting controller" name="client-ca-bundle::/etc/kubernetes/pki/ca.crt"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.041035     298 container_manager_linux.go:262] "Container manager verified user specified cgroup-root exists" cgroupRoot=[kubelet]
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.041247     298 container_manager_linux.go:267] "Creating Container Manager object based on Node Config" nodeConfig={RuntimeCgroupsName: SystemCgroupsName: KubeletCgroupsName: KubeletOOMScoreAdj:-999 ContainerRuntime: CgroupsPerQOS:true CgroupRoot:/kubelet CgroupDriver:systemd KubeletRootDir:/var/lib/kubelet ProtectKernelDefaults:false NodeAllocatableConfig:{KubeReservedCgroupName: SystemReservedCgroupName: ReservedSystemCPUs: EnforceNodeAllocatable:map[pods:{}] KubeReserved:map[] SystemReserved:map[] HardEvictionThresholds:[]} QOSReserved:map[] ExperimentalCPUManagerPolicy:none ExperimentalCPUManagerPolicyOptions:map[] ExperimentalTopologyManagerScope:container ExperimentalCPUManagerReconcilePeriod:10s ExperimentalMemoryManagerPolicy:None ExperimentalMemoryManagerReservedMemory:[] ExperimentalPodPidsLimit:-1 EnforceCPULimits:true CPUCFSQuotaPeriod:100ms ExperimentalTopologyManagerPolicy:none}
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.041383     298 topology_manager.go:133] "Creating topology manager with policy per scope" topologyPolicyName="none" topologyScopeName="container"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.041482     298 container_manager_linux.go:302] "Creating device plugin manager" devicePluginEnabled=true
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.041621     298 state_mem.go:36] "Initialized new in-memory state store"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.046475     298 kubelet.go:376] "Attempting to sync node with API server"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.046502     298 kubelet.go:267] "Adding static pod path" path="/etc/kubernetes/manifests"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.046521     298 kubelet.go:278] "Adding apiserver pod source"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.046533     298 apiserver.go:42] "Waiting for node sync before watching apiserver pods"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.047801     298 kuberuntime_manager.go:239] "Container runtime initialized" containerRuntime="containerd" version="v1.6.4" apiVersion="v1"
Jun 03 06:08:42 kind-worker kubelet[298]: W0603 06:08:42.048162     298 probe.go:268] Flexvolume plugin directory at /usr/libexec/kubernetes/kubelet-plugins/volume/exec/ does not exist. Recreating.
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.049808     298 server.go:1181] "Started kubelet"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.050110     298 server.go:150] "Starting to listen" address="0.0.0.0" port=10250
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.051353     298 server.go:410] "Adding debug handlers to kubelet server"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.052292     298 fs_resource_analyzer.go:67] "Starting FS ResourceAnalyzer"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.054984     298 volume_manager.go:289] "Starting Kubelet Volume Manager"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.055150     298 desired_state_of_world_populator.go:145] "Desired state populator starts to run"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.060374     298 kubelet.go:2344] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Jun 03 06:08:42 kind-worker kubelet[298]: W0603 06:08:42.065475     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: W0603 06:08:42.065647     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.065916     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.065676     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.065602     298 controller.go:144] failed to ensure lease exists, will retry in 200ms, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.065476     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a2f414bdf", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"Starting", Message:"Starting kubelet.", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 49784799, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 49784799, time.Local), Count:1, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:anonymous" cannot create resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: W0603 06:08:42.074690     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.074722     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.108450     298 kubelet_network_linux.go:76] "Initialized protocol iptables rules." protocol=IPv4
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.118031     298 cpu_manager.go:213] "Starting CPU manager" policy="none"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.118062     298 cpu_manager.go:214] "Reconciling" reconcilePeriod="10s"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.118078     298 state_mem.go:36] "Initialized new in-memory state store"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.118123     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), Count:1, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:anonymous" cannot create resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.118846     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), Count:1, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:anonymous" cannot create resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.119496     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), Count:1, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:anonymous" cannot create resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.121435     298 policy_none.go:49] "None policy: Start"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.122105     298 memory_manager.go:168] "Starting memorymanager" policy="None"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.122129     298 state_mem.go:35] "Initializing new in-memory state store"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.142209     298 kubelet_network_linux.go:76] "Initialized protocol iptables rules." protocol=IPv6
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.142236     298 status_manager.go:161] "Starting to sync pod status with apiserver"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.142253     298 kubelet.go:1974] "Starting kubelet main sync loop"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.142342     298 kubelet.go:1998] "Skipping pod synchronization" err="[container runtime status check may not have completed yet, PLEG is not healthy: pleg has yet to be successful]"
Jun 03 06:08:42 kind-worker kubelet[298]: W0603 06:08:42.143728     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.143759     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.155875     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.156701     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.158228     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.158288     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 156666369, time.Local), Count:2, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.159177     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 156673100, time.Local), Count:2, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.160186     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 156676503, time.Local), Count:2, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.167561     298 manager.go:610] "Failed to read data from checkpoint" checkpoint="kubelet_internal_checkpoint" err="checkpoint is not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.168372     298 eviction_manager.go:254] "Eviction manager: failed to get summary stats" err="failed to get node info: node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.168516     298 plugin_manager.go:114] "Starting Kubelet Plugin Manager"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.170515     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a36634810", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeAllocatableEnforced", Message:"Updated Node Allocatable limit across pods", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 169452560, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 169452560, time.Local), Count:1, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:anonymous" cannot create resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.256622     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.267894     298 controller.go:144] failed to ensure lease exists, will retry in 400ms, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.357209     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.359113     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.360587     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.360605     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 359073073, time.Local), Count:3, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.361579     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 359085603, time.Local), Count:3, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.362539     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 359089020, time.Local), Count:3, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.458071     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.559147     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.659772     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.669016     298 controller.go:144] failed to ensure lease exists, will retry in 800ms, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.760420     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: I0603 06:08:42.762184     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.763422     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.763408     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 762135684, time.Local), Count:4, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.764373     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 762152684, time.Local), Count:4, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.851721     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 762162681, time.Local), Count:4, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.861129     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:42 kind-worker kubelet[298]: E0603 06:08:42.961787     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: W0603 06:08:43.001316     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.001354     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.062194     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: W0603 06:08:43.107055     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.107104     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.162411     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: W0603 06:08:43.189067     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.189102     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.263551     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.364156     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.464846     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.471141     298 controller.go:144] failed to ensure lease exists, will retry in 1.6s, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:43 kind-worker kubelet[298]: W0603 06:08:43.477964     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.478003     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:43 kind-worker kubelet[298]: I0603 06:08:43.564203     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.565418     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.565527     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.565629     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 43, 564150347, time.Local), Count:5, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.566426     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 43, 564166134, time.Local), Count:5, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.567225     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 43, 564169306, time.Local), Count:5, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.665882     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.766548     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.867225     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:43 kind-worker kubelet[298]: E0603 06:08:43.967903     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.069010     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.169775     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.270549     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.371172     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.471954     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.572706     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.673396     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.774053     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.874791     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:44 kind-worker kubelet[298]: E0603 06:08:44.975506     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.072175     298 controller.go:144] failed to ensure lease exists, will retry in 3.2s, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.076238     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: I0603 06:08:45.166793     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.169403     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 45, 166759258, time.Local), Count:6, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.169512     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.170798     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 45, 166764432, time.Local), Count:6, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.171741     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 45, 166767432, time.Local), Count:6, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.176881     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.277542     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.378249     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.478869     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.579790     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.680439     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: W0603 06:08:45.748859     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.748896     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.781423     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: W0603 06:08:45.793926     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.793967     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.882532     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:45 kind-worker kubelet[298]: W0603 06:08:45.909209     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.909250     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:anonymous" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
Jun 03 06:08:45 kind-worker kubelet[298]: E0603 06:08:45.983654     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.084011     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: W0603 06:08:46.157667     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.157727     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.184935     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.285569     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.386167     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.486837     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.587968     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.688586     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.789233     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.889876     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:46 kind-worker kubelet[298]: E0603 06:08:46.990483     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.090612     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.169875     298 kubelet.go:2344] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.191015     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.291693     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.392275     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.492864     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.593561     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.694098     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.794743     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.895313     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:47 kind-worker kubelet[298]: E0603 06:08:47.996105     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.097127     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.198133     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.274323     298 controller.go:144] failed to ensure lease exists, will retry in 6.4s, error: leases.coordination.k8s.io "kind-worker" is forbidden: User "system:anonymous" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.298417     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: I0603 06:08:48.370880     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.372858     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333d9207", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientMemory", Message:"Node kind-worker status is now: NodeHasSufficientMemory", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116649479, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 48, 370837704, time.Local), Count:7, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333d9207" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.372953     298 kubelet_node_status.go:92] "Unable to register node with API server" err="nodes is forbidden: User \"system:anonymous\" cannot create resource \"nodes\" in API group \"\" at the cluster scope" node="kind-worker"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.373875     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dacd9", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasNoDiskPressure", Message:"Node kind-worker status is now: NodeHasNoDiskPressure", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116656345, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 48, 370850111, time.Local), Count:7, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dacd9" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.374934     298 event.go:267] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"kind-worker.16f5075a333dbbf6", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ZZZ_DeprecatedClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Node", Namespace:"", Name:"kind-worker", UID:"kind-worker", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"NodeHasSufficientPID", Message:"Node kind-worker status is now: NodeHasSufficientPID", Source:v1.EventSource{Component:"kubelet", Host:"kind-worker"}, FirstTimestamp:time.Date(2022, time.June, 3, 6, 8, 42, 116660214, time.Local), LastTimestamp:time.Date(2022, time.June, 3, 6, 8, 48, 370853668, time.Local), Count:7, Type:"Normal", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "kind-worker.16f5075a333dbbf6" is forbidden: User "system:anonymous" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.398999     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.499570     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.600416     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.701166     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.801948     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:48 kind-worker kubelet[298]: E0603 06:08:48.902565     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.003240     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: W0603 06:08:49.081490     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.081527     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.RuntimeClass: failed to list *v1.RuntimeClass: runtimeclasses.node.k8s.io is forbidden: User "system:anonymous" cannot list resource "runtimeclasses" in API group "node.k8s.io" at the cluster scope
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.103558     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.204455     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.305049     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.405671     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.506254     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.607170     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.708284     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.808391     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:49 kind-worker kubelet[298]: E0603 06:08:49.908877     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.009446     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: W0603 06:08:50.012298     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.012331     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Node: failed to list *v1.Node: nodes "kind-worker" is forbidden: User "system:anonymous" cannot list resource "nodes" in API group "" at the cluster scope
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.109911     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.210888     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.311421     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.412219     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.512380     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.613079     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.713633     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.814209     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:50 kind-worker kubelet[298]: E0603 06:08:50.914706     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.015304     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.116302     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.217024     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.317673     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.418217     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.518766     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.619881     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: W0603 06:08:51.662537     298 reflector.go:324] vendor/k8s.io/client-go/informers/factory.go:134: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.662571     298 reflector.go:138] vendor/k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: services is forbidden: User "system:anonymous" cannot list resource "services" in API group "" at the cluster scope
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.720772     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.821415     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:51 kind-worker kubelet[298]: E0603 06:08:51.921960     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.022522     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: I0603 06:08:52.029692     298 transport.go:135] "Certificate rotation detected, shutting down client connections to start using new credentials"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.124947     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.169301     298 eviction_manager.go:254] "Eviction manager: failed to get summary stats" err="failed to get node info: node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.192334     298 kubelet.go:2344] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.225808     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.328161     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.428704     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.485228     298 csi_plugin.go:297] Failed to initialize CSINode: error updating CSINode annotation: timed out waiting for the condition; caused by: nodes "kind-worker" not found
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.529817     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.630256     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.730604     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.831134     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:52 kind-worker kubelet[298]: E0603 06:08:52.931431     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.032266     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.133278     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.234147     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.335160     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.435694     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.535771     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.636308     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.667040     298 csi_plugin.go:297] Failed to initialize CSINode: error updating CSINode annotation: timed out waiting for the condition; caused by: nodes "kind-worker" not found
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.737321     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.838127     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:53 kind-worker kubelet[298]: E0603 06:08:53.938365     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.039184     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.139655     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.240715     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.341704     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.442230     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.543224     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.643809     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.677675     298 nodelease.go:49] "Failed to get node when trying to set owner ref to the node lease" err="nodes \"kind-worker\" not found" node="kind-worker"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.743915     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: I0603 06:08:54.774020     298 kubelet_node_status.go:70] "Attempting to register node" node="kind-worker"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.844285     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:54 kind-worker kubelet[298]: E0603 06:08:54.945253     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.045952     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: I0603 06:08:55.070259     298 kubelet_node_status.go:73] "Successfully registered node" node="kind-worker"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.146990     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.247613     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.348272     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.448844     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.549661     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.650221     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.750564     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.851156     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:55 kind-worker kubelet[298]: E0603 06:08:55.951706     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.052739     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.153667     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.254688     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.355263     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.455855     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.556647     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.657091     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.757481     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.857607     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:56 kind-worker kubelet[298]: E0603 06:08:56.958092     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.058913     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.159694     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.193545     298 kubelet.go:2344] "Container runtime network not ready" networkReady="NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.260668     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.361949     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.462503     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
Jun 03 06:08:57 kind-worker kubelet[298]: E0603 06:08:57.563491     298 kubelet.go:2419] "Error getting node" err="node \"kind-worker\" not found"
